arwen/contexts/async.go

```diff
@@ -705,14 +678,17 @@ func (context *asyncContext) findGroupByID(groupID string) (int, bool) {
 
 func computeDataLengthFromArguments(function string, arguments [][]byte) int {
```

Comment:
why do we need this ?

Reply:
and why do we need so exactly ?

Reply:
I think the cost is already taken out when making asyncCall - there is a base cost plus data copy per byte, which should be enough.

Reply:
It's the cost of passing the returned data from the destination call into the arguments of the callback. It must match the cost of the `Data` field of a SCR would have, for cross-shard calls.

Reply:
it could be simply 2xdataLen :D